docs/source/markdown/podman-kube-play.1.md

```diff
@@ -1,15 +1,15 @@
-% podman-play-kube(1)
+% podman-kube-play(1)
```

Comment:
I would like to keep the `play-kube` man page but symlink it to `kube-play`.  Just to make the life of existing users easier.

Reply:
They can be linked together, see the links subdir.